BioinformaticsScripts
=====================

**TrinBlast:**

After you have run a BLAST against your contigs which were assembled with Trinity, this program takes your the text file results from this BLAST and parses it and the Trinity.fasta output to create a CSV file containing the contig identifier, length of the contig, description of the closest matching sequence and the sequence itself.

**FindDigest:**

Automatically retrieve a sequence and then find the fragment lengths generated by a restriction enzyme digest

**MassCDD:**

Takes all the sequences in a fasta file and runs them against the NCBI Conserved Domain Database

**SafeBlast:**

Runs a Blast against the NCBI database using BioPython qblast, without crashing due to timeout/loss of connection
